digitalpublicgoods/community-health-toolkit.json

```diff
@@ -0,0 +1,96 @@
+      ],
+      "thirdPartyDataSharing": "No",
+      "dataSharingCircumstances": [
+        ""
+      ],
+      "ensurePrivacySecurity": "Yes",
+      "privacySecurityDescription": "Privacy policy is different per app, it is the responsibility of the Data Controlling Entity to establish such policies. That the CHT community grants Data Controlling Entities the freedom to establish their own policies is essential to the terms of our OSI approved open source license. The following are the steps taken to prevent adverse impact related to PII: Encryption, isolation and restrictive permissions"
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "No",
+      "type": "",
+      "contentFilter": "No",
+      "policyGuidelinesDocumentationLink": "",
+      "illegalContentDetection": "No",
+      "illegalContentDetectionMechanism": ""
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "Yes",
+      "addressSafetySecurityUnderageUsers": "Yes",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [
+        "Most CHT apps are designed specifically for use by health workers; user accounts are provided by organizations such that access is not available to the general public. In CHT apps that support messaging with patients, the relevant patients are enrolled and consented into the process by a trained health worker. To our knowledge there have not been any deployments of CHT apps in which minors were enrolled in use of the system."
+      ],
+      "griefAbuseHarassmentProtection": "Yes",
+      "harassmentProtectionSteps": [
+        "One of the four core principles of the CHT community is that we are committed to designing CHT systems in a human-centered manner. In our view, this is not merely a buzzword or a statement of ethical priorities. Rather, it is a set of rigorous methods that work within a flexible yet disciplined process to prioritize people’s needs and concrete experiences in the design of digital systems. An essential feature of human-centered design is that it prioritizes the dignity and agency of end users. It is also explicitly iterative and takes responsibility for making adaptations to address any potential unintended effects of a system - such as a situation in which health workers might experience grief or abuse from a peer or supervisor via their digital tool. We have conducted extensive research and members of our team are considered global authorities on the topic of human-centered design for global health, we consider this approach to be a thorough and substantive means of addressing lived experiences of harassment and abuse in health systems. More about CHT principles and our human centered design approach in the following links: \nhttps://communityhealthtoolkit.org/principles \nhttps://www.tandfonline.com/doi/full/10.1080/02681102.2019.1667289"
+      ]
+    }
+  },
+  "locations": {
+    "developmentCountries": [
```

Comment:
I know it's not a required field, but it would be really nice to have some countries listed here. Thoughts? @nathanbaleeta

Reply:
Resolved in commit 08768cb